upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -231,3 +231,9 @@ There you can find links to upgrade notes for other versions too.
 
 - add automatic string trimming and new Password type in frontend API ([#2127](https://github.com/shopsys/shopsys/pull/2127))
     - see #project-base-diff to update your project
+
+- use product by uuid from Elasticsearch in FE API (([#2131](https://github.com/shopsys/shopsys/pull/2131)))
```

Comment:
I am not sure if I like this line. Would not do something like this a bit better?

```
- FE API provides single product from Elasticsearch (([#2131](https://github.com/shopsys/shopsys/pull/2131)))
```

Reply:
I don't think so. 
in upgrade notes should be instructions. I tried to rephrase the line in e365b3867